azure/delete_resource_group.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
The create script uses dashes in the name and the delete script uses underscores. Should maybe standardize on dashes.